test/pkg/debug/collector.go

```diff
@@ -142,13 +142,17 @@ func PrintTestTimes(times map[string][]TimeInterval) map[string][][]string {
 }
 
 // WriteTimestamps will create a temp directory and a .csv file for each suite test
-func WriteTimestamps(path string, timestamps *TimeIntervalCollector) error {
-	directory, err := os.MkdirTemp("/tmp", "")
-	if err != nil {
-		return err
+// If the OUTPUT_DIR environment variable is set, we'll print the csvs to that directory.
+func WriteTimestamps(outputDir string, timestamps *TimeIntervalCollector) error {
+	var err error
+	if outputDir == "" {
```

Comment:
If the output dir doesn't exist, do we just want to create the whole path on behalf of the user, regardless of whether it's using the default or not?

Reply:
I think we should expect that a user has preprovisioned the directory, and wouldn't expect us to recursively create the directories for them.